### DISCUSSION TITLE
Render the has_one/has_many associations on the edit page

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#755] [Feature requests]
We have some feedback from a content team (non-technical users of the app) that they would like the has_many associations in the edit page.
```
- It seems that they quite often dont understand the difference between "edit" and "view".
So when they click on edit, they are wondering where the "has_many" resources are and ask me then. After some time you get more used to it, but it still feels a bit strange to them
```

**Comment by Paul-Bob** [comment]
`field :user, as: :has_one, show_on: :edit`
`field :users, as: :has_many, show_on: :edit`

FUN FACT: This was my first avo PR https://github.com/avo-hq/avo/pull/880 🥲
